libnetwork_test.go

```diff
@@ -97,7 +97,7 @@ func getPortMapping() []types.PortBinding {
 }
 
 func TestNull(t *testing.T) {
-	network, err := createTestNetwork("null", "testnetwork", options.Generic{})
+	network, err := createTestNetwork("null", "testnull", options.Generic{})
```

Comment:
I can see that the network names are used as unique values across the tests to make sure they dont collide with each other across tests (due to the recent change to the libnetwork_test with a single controller for all the tests).
But, I think the correct approach is to do proper cleanup of these states on each test to make sure these tests doesnt impact the next. As an example, i recently added `TestDuplicateEndpoint` to handle a similar situation. Pls take a look at that.